server/service/osquery_utils/queries.go

```diff
@@ -1040,6 +1040,18 @@ SELECT
   '' AS installed_path
 FROM rpm_packages
 UNION
+SELECT
+  name AS name,
+  version AS version,
+  '' AS extension_id,
+  '' AS browser,
+  'fleetd_pacman_packages' AS source,
+  '' AS release,
+  '' AS vendor,
+  arch AS arch,
+  '' AS installed_path
+FROM fleetd_pacman_packages
+UNION
```

Comment:
Added query cost is negligible on non-Arch linux hosts, right? (because there's no `/usr/bin/pacman`)

Reply:
Correct, if the binary doesn't exist it just immediately returns no results